apps/bot/src/tweetRanking/jobs/getRuleFromConfig.ts

```diff
@@ -11,14 +11,12 @@ const getRuleFromConfig = (varName: string) => {
 
   const [hour, minute] = hourTimeString.split(':');
 
-  return new RecurrenceRule(
-    null,
-    null,
-    null,
-    null,
-    parseInt(hour),
-    parseInt(minute),
-  );
+  const rule = new RecurrenceRule();
+  rule.hour = parseInt(hour, 10);
+  rule.minute = parseInt(minute, 10);
+  rule.tz = 'America/Sao_Paulo';
```

Comment:
I think you can use

```ts
new RecurrenceRule({
  hour: parseInt(hour, 10),
  minute: parseInt(minute, 10),
  tz: 'America/Sao_Paulo',
})```

Reply:
didn't worked, btw, thanks for the suggestion

```typescript
new RecurrenceRule({
  hour: parseInt(hour, 10),
  minute: parseInt(minute, 10),
  tz: 'America/Sao_Paulo',
})

//Argument of type '{ hour: number; minute: number; tz: string; }' is not assignable to parameter of type 'RecurrenceSegment'.
//Object literal may only specify known properties, and 'hour' does not exist in type 'Range | Recurrence[]'.ts(2345)
```

Reply:
Sure, nice